Source/bmalloc/bmalloc/Environment.cpp

```diff
@@ -148,6 +148,12 @@ static bool isRunningOnValgrind()
     return false;
 }
 
+static bool isSystemHeapForceDisabled()
+{
+    const char* value = getenv("WebKitSystemHeapOverrideEnablement");
+    return value ? !atoi(value) : false;
+}
+
 #if BUSE(CHECK_NANO_MALLOC)
```

Comment:
An aside but I don't think Nano malloc exists anymore we can probably get rid of this.

Reply:
Some complications, will look into it for another patch